test/addon_spec.rb

```diff
@@ -171,8 +170,8 @@
   describe "#selectable?" do
     let(:addons) do
       Registration::Addon.find_all(double(
-          get_addon_list:     [addon_with_child_generator],
-          activated_products: []
+                                     get_addon_list:     [addon_with_child_generator],
+                                     activated_products: []
```

Comment:
yep, it is indended to align method call, so your one with double is nicer. This is just auto-correct.